### PR TITLE
chore: update wording from 'update' to 'replace' and add GitHub Actions COMPASS-4525

### PR DESCRIPTION
**Pull request by rose-m**
## Description
Updates the button in the footer of the JSON document edit view to say _Replace_ instead of _Update_.

image

**This also replaces TravisCI with GitHub Workflows.**

### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added

## Motivation and Context
- [x] Bugfix
- [ ] New feature
- [ ] Dependency update
- [x] Misc

## Types of changes
- [ ] *Backport Needed*
- [x] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by addaleax — APPROVED** [review]

**Review by mcasimir — APPROVED** [review]

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        with:
+          node-version: 12.4.0
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
```
Comment (mcasimir) [inline .github/workflows/run-tests.yml:33]: nice trick

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  pull_request:
+
+jobs:
+  check-and-test:
+    name: Check and Test
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        mongoVersion: [stable, unstable]
```
Comment (mcasimir) [inline .github/workflows/run-tests.yml:15]: I'm surprised that this work

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (rose-m) [reply .github/workflows/run-tests.yml:33]: (kudos to their official documentation :D)

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (rose-m) [reply .github/workflows/run-tests.yml:15]: It's the environment variable - `MONGODB_VERSION` (https://github.com/mongodb-js/compass-crud/pull/317/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R18). At least that's what it seems from the previous config...

**Review by addaleax — APPROVED** [review]
:+1:

**Pull request merged**